### DISCUSSION TITLE
Why was `chars` deprecated?

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [dicebear/dicebear#444] [Q&A] [answered]
Why was `chars` deprecated?
https://www.dicebear.com/styles/initials/#options-chars

![Image](https://github.com/user-attachments/assets/8e5a1bc7-bc3d-47d5-88cf-cfe92cac000a)

**Comment by atumas-bananamilk** [comment]
Any answer? Thanks

**Comment by FlorianKoerner** [comment, marked as answer]
With DiceBear 10, all avatar styles are described using a JSON file. This has the advantage that more programming languages can be easily supported. At the same time, however, it also brings with it a certain restriction of flexibility when it comes to such specific options.

However, I have an idea how we could continue to support this option. But, as this has not yet been implemented, the future status of the option is uncertain. I will definitely take a closer look at the issue. My goal would be that we don't have to rely on any option.

**Reply by atumas-bananamilk** [reply]
Thanks for reply.
However, it's quite become popular to use 1-letter, at least please make it default (default = 2 now).
Obviously the best option is to allow `chars` back.
